### PR TITLE
Release v1.7.2

### PR DESCRIPTION
**Pull request by jmrodri**
Signed-off-by: jesus m. rodriguez <jesusr@redhat.com>

**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by estroz — CHANGES_REQUESTED** [review]
Needs to target v1.7.x

**Comment by jmrodri** [comment]
> Needs to target v1.7.x

:man_facepalming:

File: website/content/en/docs/installation/_index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ export OS=$(uname | awk '{print tolower($0)}')
 Download the binary for your platform:
 
 ```sh
-export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.7.1
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/v1.7.2
```
Comment (estroz) [inline website/content/en/docs/installation/_index.md:39]: Update script needs fixing
```suggestion
export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.7.2
```

**Review by estroz — APPROVED** [review]
LGTM after fixing link

File: website/content/en/docs/upgrading-sdk-version/v1.7.2.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: v1.7.2
+weight: 998992998
+---
+
+There are no migrations for this release! :tada:
```
Comment (asmacdo) [inline website/content/en/docs/upgrading-sdk-version/v1.7.2.md:6]: :tada:

**Pull request merged**